src/Header.css

```diff
@@ -0,0 +1,38 @@
+.header {
+  height: 20%;
+  width:100%;
+  position:relative;
+}
+
+div div div .button{
```

Comment:
Nested css here is pretty bad practice, i'd like to see another way to avoid this